js/build.xml

```diff
@@ -863,14 +816,9 @@ limitations under the License.
 	<target name="test.core" depends="test.compiled,test.uncompiled"
 			description="run the tests">
 	</target>
-	<target name="core.junit.report" description="Generate a report on the junit tests">
-		<mkdir dir="${build.output.reports}/junit"/>
-	</target>
 	<!-- target name="test" depends="test.core"/ -->
 	<target name="test.only" depends="test.all" description="Run all tests"/>
-	<target name="reports.only" depends="core.junit.report" description="Generate reports on previously run tests"/>
-	<target name="reports" depends="test.only,reports.only" description="Run all tests, then generate reports on the results"/>
-	<target name="test" depends="reports" description="Run all tests and build all reports"/>
+	<target name="test" description="Run all tests and build all reports"/>
```

Comment:
test doesn't depend on anything any more... so if you do "ant test" it won't do anything.